docs/iot-gateway/config/opc-ua.md

```diff
@@ -125,61 +158,163 @@ This part of configuration will look like:
 
 | **Parameter**   | **Default value**           | **Description**                                                                   |
 |:-|:-|-
-| key             | **humidity**                    | Tag, that will interpreted as telemetry for ThingsBoard platform instance.        |
-| path            | **${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Humidity}**           | Name of the variable in the OPC-UA object, uses for looking the value in some variable. ** \* ** |
+| key             | **temperature °C**                                                             | Tag, that will interpreted as telemetry for ThingsBoard platform instance.        |
+| path            | **${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Temperature}**   | Name of the variable in the OPC-UA object, uses for looking the value in some variable. ** \* ** |
 |---
 
 ** \* ** You can put here some expression for search like:
-1. Full path to node - **${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Humidity}**
-2. Relative path from device object - **${TemperatureAndHumiditySensor\\.Humidity}** 
-3. Some regular expression to search - **${Root\\.Objects\\.Device\\d\*\\.TemperatureAndHumiditySensor\\.Humidity}**
+1. Full path to node - **${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Temperature}**
+2. Relative path from device object - **${TemperatureAndHumiditySensor\\.Temperature}** 
+3. Some regular expression to search - **${Root\\.Objects\\.Device\\d\*\\.TemperatureAndHumiditySensor\\.Temperature}**
 4. Namespace identifier and node identifier - **${ns=2;i=5}**
 
 This part of configuration will look like:  
 
+```json
+        "timeseries": [
+          {
+            "key": "temperature °C",
+            "path": "${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Temperature}"
+          }
+        ],
+```
+
+<br>
+**Let's look an example.**
+
+Replace "path" value to the "NodeId" value, relative path from device object or Display Name identifier, taken from our test server.
+<br>
+
+{:refdef: style="text-align: left;"}
+![image](/images/gateway/opc-ua-simulation-server-4.png)
+{: refdef}
+
+In this example, the **timeseries** section would look like this:
+
 ```json
         "timeseries": [
           {
             "key": "humidity",
-            "path": "${Root\\.Objects\\.Device1\\.TemperatureAndHumiditySensor\\.Humidity}"
+            "path": "${Counter}"
+          },
+          {
+            "key": "pressure",
+            "path": "Root\\.Objects\\.Simulation\\.Triangle"
+          },
+          {
+            "key": "temperature °C",
+            "path": "${ns=3;i=1002}"
           }
         ],
 ```
 
+{:refdef: style="text-align: left;"}
+![image](/images/gateway/opc-ua-configuration-4.png)
+{: refdef}
 
-#### Subsection "rpc_methods"
+In the ThingsBoard instance, you must see the **latest telemetry** in your device:
```

Comment:
The same as the comment about attributes

Reply:
+

Reply:
replaced